app/routes.py

```
@@ -0,0 +1,40 @@
+        if not os.path.exists(request.repo_path):
+            raise HTTPException(status_code=400, detail="Repository path does not exist")
+        
+        # Use repo_name from request or default to directory name
+        repo_name = request.repo_name or os.path.basename(request.repo_path)
+        
+        # Run comparison
+        result = comparator.compare_commits(
+            request.repo_path,
+            request.old_commit,
+            request.new_commit,
+            repo_name
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)